osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/dlmsobjectconfig/DlmsObjectConfigSmr50.java

```diff
@@ -283,6 +284,10 @@
         new DlmsObject(CLIENT_SETUP_MBUS, InterfaceClass.MBUS_CLIENT.id(), "0.<c>.24.1.0.255");
     objectList.addAll(Arrays.asList(readMBusStatus, clientSetupMBus));
 
+    // Alarm registers
+    final DlmsObject alarmRegisters1 = new DlmsData(ALARM_REGISTER_1, "0.0.97.98.0.255");
+    objectList.addAll(Arrays.asList(alarmRegisters1));
+
```

Comment:
I would prefer to add the alarmRegister to the section with abstract objects at the beginning of this file. In the SMR documentation the alarmRegister is just above the alarmFilter.  The same applies to the DSMR4 configuration.

Reply:
Moved to abstract section